scripts/metadata_comparison/metadata_comparison/extractor.py

```diff
@@ -22,9 +22,10 @@
 import requests
 from google.cloud import storage
 import google.auth
-from googleapiclient.discovery import build as google_client_build
 import logging as log
 from metadata_comparison.lib.argument_regex import *
+from metadata_comparison.lib.operation_ids import *
```

Comment:
`*` imports import everything. Not just everything from the file, but also all its imports. Basically all names that are available in that namespace. In general it is advisable to explicitly only import the functions that are needed. But in case it is 10+ functions it gets a bit impractical, and then a `*` might be better.